retroshare-gui/src/gui/notifyqt.cpp

```diff
@@ -89,6 +89,10 @@ void NotifyQt::SetDisableAll(bool bValue)
 	}
 }
 
+NotifyQt::~NotifyQt()
```

Comment:
use `= default;` for empty destructors as the compiler can optimize better those

Reply:
Or did you mean to unregister the event handler here?

Reply:
actually it's a remaining of the previous use of the handler in notifyQt but I moved it to Feed